root/static/scripts/guess-case/utils.js

```diff
@@ -206,19 +206,22 @@ export function titleString(gc, is, forceCaps) {
      * we got an 'round (Around = 'round), lowercase
      * we got a 'mon (Come on = C'mon), lowercase
      */
-  } else if (isApostrophe(gc.i.getPreviousWord()) &&
+  } else if (gc.mode.fixEnglishApostrophes() &&
+      isApostrophe(gc.i.getPreviousWord()) &&
       lc.match(/^(?:s|round|em|ve|ll|d|cha|re|til|way|all|mon)$/i)) {
```

Comment:
This style of indentation makes it slightly harder for me to tell which lines are part of the condition and which are part of the `if` block. I like to follow our general brace style for multi-line conditions, so I'm recommending that unless you strongly prefer the current way. :)
```suggestion
  } else if (
    gc.mode.fixEnglishApostrophes() &&
    isApostrophe(gc.i.getPreviousWord()) &&
    lc.match(/^(?:s|round|em|ve|ll|d|cha|re|til|way|all|mon)$/i)
  ) {
```